pipeline/transformer/transformer_miro/src/test/scala/uk/ac/wellcome/platform/transformer/miro/services/MiroVHSRecordReceiverTest.scala

```diff
@@ -94,13 +92,12 @@ class MiroVHSRecordReceiverTest
     }
   }
 
-  it("returns a failed future if there's no MiroMetadata") {
+  // It's not possible to store a record without metadata with the HybridStore
+  // used in these tests
+  ignore("returns a failed future if there's no MiroMetadata") {
```

Comment:
A new test which uses a store with failing get might capture something similar to what this test is doing. Thoughts?

Reply:
Yes 👍 no need to do that to merge this though.